### ISSUE TITLE
updating coordinates of a shape

### ISSUE THREAD

**Issue by MBottinIUT** [adafruit/Adafruit_CircuitPython_Display_Shapes#61]
hello, I asked the question in the forum and was advised to open an issue here.
On several occasions, I would have liked to be able to modify the coordinates of a shape during the execution of a code (for example, move a line on the screen).
But I can't change a shape's attributes once it's been instantiated.

Would it be possible to add the ability to update the position of a line (or another shape) without re-creating the constructor ?

Thanks

**Comment by stratosgear** [comment]
Yes!  I have to create some kind of progress bar, or line that changes length, and I fail to animate my shapes, after they have been created.

I would appreciate if we could get access to the x0/y0, x1/y1 attributes of the shapes!

Thanks!

**Comment by bablokb** [comment]
The position of a shape is given by `x` and `y` and these are attributes which you can change. In the examples there is an example with an animated circle that does exactly this.
